gprofiler/main.py

```diff
@@ -113,7 +118,13 @@ def __init__(
         self._client = client
         self._state = state
         self._remote_logs_handler = remote_logs_handler
+        self._collect_metrics = collect_metrics
+        self._collect_metadata = collect_metadata
         self._stop_event = Event()
+        self._static_metadata: Optional[Metadata] = None
```

Comment:
Don't need `Optional` here (since we're setting to `None`)

Reply:
We have both versions in the code :(

Reply:
k